.github/workflows/large_vote_ci.yml

```diff
@@ -16,6 +16,15 @@ jobs:
     name: Brownie fork LARGE tests
     runs-on: [protocol-heavy-runners]
     timeout-minutes: 150
+
+    services:
+      hardhat-node:
+        image: ghcr.io/lidofinance/hardhat-node:2.22.17
+        ports:
+          - 8545:8545
+        env:
+          ETH_RPC_URL: "https://mainnet.infura.io/v3/${{ secrets.WEB3_INFURA_PROJECT_ID }}"
```

Comment:
Can be just `INFURA_TOKEN` no need to provide full RPC
https://github.com/lidofinance/hardhat-node?tab=readme-ov-file#run